doc/api/util.md

```diff
@@ -2492,6 +2492,18 @@ const util = require('util');
 util.log('Timestamped message.');
 ```
 
+
+### `util.toUSVString(string)`
```

Comment:
just realized... isn't this in the wrong section? the above functions are all deprecated

Reply:
You are right. I also did not notice.

Reply:
https://github.com/nodejs/node/pull/39840